app/controllers/orders_controller.rb

```diff
@@ -326,7 +314,7 @@ def index
   # Group into chunks by the group id. If group_id is nil, then it should get its
   # own chunk.
   def grouped_order_details
-    @order.order_details.order(:group_id).slice_when do |before, after|
+    @order.order_details.sort_by(&:group_id).slice_when do |before, after|
```

Comment:
Oops. You are correct. Did not mean to commit this. This was related to a failed experiment I was trying.